### PR TITLE
Add labels option

### PR DESCRIPTION
**Pull request by zyclonite**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -293,6 +293,17 @@ config {
 
 * **tty** - (Optional)  true or false (default). Allocate a pseudo-TTY for the container.
 
+* **labels** - (Optional)  Set labels on the container.
+
+```
+config {
+  options = [
```
Comment (towe75) [inline README.md:300]: @zyclonite : should be "labels".

File: config.go [hunk 2]
```diff
@@ -53,6 +53,7 @@ var (
 		"image":              hclspec.NewAttr("image", "string", true),
 		"init":               hclspec.NewAttr("init", "bool", false),
 		"init_path":          hclspec.NewAttr("init_path", "string", false),
+		"labels":             hclspec.NewAttr("labels", "list(map(string))", false),
```
Comment (towe75) [inline config.go:56]: Any reason why it's not just a map(string) ?

**Review by towe75 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zyclonite) [reply README.md:300]: ups, you are right

File: config.go (hunk 2 above)
Comment (zyclonite) [reply config.go:56]: i did look that up from the docker driver... i was not sure as well, never used hcl before

Comment (zyclonite) [reply config.go:56]: https://github.com/hashicorp/nomad/blob/bdf2555b385f547cbf613d1636eac103a1c58ce8/drivers/docker/config.go#L361

Comment (towe75) [reply config.go:56]: IC. But the docker config example shows no "[" , "]" list elements, just a map. Can you try to run you code against a simplified hcl without the brackets?

Comment (zyclonite) [reply config.go:56]: sure, works as well

**Comment by towe75** [comment]
@zyclonite the hcl test and examples look certainly nicer now, thank you. 

Maybe you can write a simple driver_test, too? Check the TestPodmanDriver_Tty for a good example. It boils down to see if the attached labels are visible in the inspected container. After that we're good to go.

**Comment by zyclonite** [comment]
@towe75 driver test added

**Review by towe75 — APPROVED** [review]
@zyclonite  LGTM!

**Pull request merged**